content/docs/railway-wiki/glossary/_index.fr.md

```diff
@@ -124,4 +126,4 @@ description: Glossaire thématique d'OSRD et du ferroviaire
 - **VS** : Voie de service
 - **VU** : Voie unique
 - **VUT** : Voie unique temporaire
-- **VUTP** : Voie unique temporaire à caractère permanent
\ No newline at end of file
+- **VUTP** : Voie unique temporaire à caractère permanent
```

Comment:
I can't seem to notice the change here, was there a missing newline or something?